src/main/java/com/minecolonies/coremod/commands/generalcommands/CheckForAutoDeletesCommand.java

```diff
@@ -0,0 +1,114 @@
+package com.minecolonies.coremod.commands.generalcommands;
+
+import com.minecolonies.api.configuration.Configurations;
+import com.minecolonies.coremod.colony.Colony;
+import com.minecolonies.coremod.colony.ColonyManager;
+import com.minecolonies.coremod.commands.AbstractSingleCommand;
+import net.minecraft.command.CommandException;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.text.ITextComponent;
+import net.minecraft.util.text.Style;
+import net.minecraft.util.text.TextComponentString;
+import net.minecraft.util.text.TextFormatting;
+import net.minecraft.util.text.event.ClickEvent;
+import org.jetbrains.annotations.NotNull;
+
+import javax.annotation.Nullable;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class CheckForAutoDeletesCommand extends AbstractSingleCommand
+{
+
+    public static final String DESC = "check";
+
+    /**
+     * Initialize this SubCommand with it's parents.
+     *
+     * @param parents an array of all the parents.
+     */
+    public CheckForAutoDeletesCommand(@NotNull final String... parents)
+    {
+        super(parents);
+    }
+
+    @Override
+    public boolean canRankUseCommand(@NotNull final Colony colony, @NotNull final EntityPlayer player)
+    {
+        return false;
+    }
+
+    @Override
+    public void execute(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, @NotNull final String... args) throws CommandException
+    {
+        if (sender instanceof EntityPlayer && !isPlayerOpped(sender))
+        {
+            sender.sendMessage(new TextComponentString("Must be OP to use command"));
+            return;
+        }
+        else if (sender instanceof TileEntity)
+        {
+            return;
+        }
+
+        final List<Colony> colonies = ColonyManager.getColonies();
+
+        final List<Colony> coloniesToDelete = new ArrayList<>();
+
+        for (int index = 0; colonies.size() - 1 >= index; index++)
+        {
+            final Colony colony = colonies.get(index);
+
+            if (colony.canBeAutoDeleted() && Configurations.gameplay.autoDeleteColoniesInHours != 0 && colony.getLastContactInHours() >= Configurations.gameplay.autoDeleteColoniesInHours)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 187 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS00103)